server/entity/action.go

```diff
@@ -41,6 +41,9 @@ type PickedUpAction struct {
 	action
 }
 
+// FireworkParticleAction is a world.EntityAction that makes a Firework rocket display an explosion particle.
+type FireworkParticleAction struct{ action }
```

Comment:
This is a bit of a poor name given fireworks are essentially all particles. Maybe something like `Firework{Explode/Explosion}Action`would be better?

Reply:
Renamed.